overlay_gl/overlay_gl.pro

```diff
@@ -1,26 +1,60 @@
-# Simple project file to build overlay with same compilers
-# as Qt used.
+# Overlay payload for Unix-like systems and OS X.
+
+include(../compiler.pri)
 
 TEMPLATE = lib
-CONFIG -= qt
+CONFIG -= qt gui
 CONFIG *= debug_and_release
-TARGET = mumble$(TARGET_ADD)
 VERSION = 1.3.0
 SOURCES = overlay.c
-LIBS *= -lrt -ldl
-QMAKE_CFLAGS *= -fvisibility=hidden $(CFLAGS_ADD)
-QMAKE_LFLAGS -= -Wl,--no-undefined
-QMAKE_LFLAGS *= $(LFLAGS_ADD)
-equals(QMAKE_LINK,g++) {
-  message(Overriding linker)
-  QMAKE_LINK = gcc
-  QMAKE_LINK_SHLIB = gcc
+
+CONFIG(static) {
+	CONFIG -= static
```

Comment:
It was in the OS X overlay.

We build with CONFIG+=static on OS X.

However, we want the overlay to be a shared library, not a static one. So we remove it.